pub_sub/go/sdk/order-processor/app.go

```diff
@@ -18,7 +19,13 @@ var sub = &common.Subscription{
 }
 
 func main() {
-	s := daprd.NewService(":6001")
+	//read app-port passed thru command line
```

Comment:
```suggestion
	// read the app-port from the environment variable
```
However, it seems unnecessary.